perllib/FixMyStreet/DB/ResultSet/Problem.pm

```diff
@@ -172,6 +172,10 @@ sub around_map {
             latitude => { '>=', $p{min_lat}, '<', $p{max_lat} },
             longitude => { '>=', $p{min_lon}, '<', $p{max_lon} },
     };
+
+    my $interval = $p{report_age};
+    $q->{confirmed} = { '>=', \"current_timestamp-'$interval'::interval" } if
```

Comment:
Been thinking about this some more, and I do think it needs to adjust to sort order. For example, if I select sort by recently updated, I expect a report updated yesterday to appear, even if it was created 2 years ago.
I think it should work as follows:
* sort order newest/oldest/most commented, use confirmed;
* sort order recently updated/least recently updated, use lastupdate.

I think that makes sense?